common/viewer.cpp

```diff
@@ -567,16 +769,19 @@ namespace rs2
     viewer_model::viewer_model()
             : ppf(*this), 
               synchronization_enable(true),
-              zo_sensors(0)
+              zo_sensors(0),
+              frameset_alloc(this)
     {
         syncer = std::make_shared<syncer_model>();
         reset_camera();
         rs2_error* e = nullptr;
         not_model.add_log(to_string() << "librealsense version: " << api_version_to_string(rs2_get_api_version(&e)) << "\n");
     
         update_configuration();
-
+        
         check_permissions();
+        export_model exp_model = export_model::make_exporter("PLY", ".ply", "Polygon File Format (PLY)\0*.ply\0");
```

Comment:
Can be named "PLY Export" or "Export to PLY" for readability

Reply:
I think this is the string that will appear in the UI as the tab name (so needs to be short)